.storybook/preview.js

```diff
@@ -34,7 +34,8 @@ export const parameters = {
         'AlternativeVerticals',
         'SpellCheck',
         'ResultsCount',
-        'LocationBias'
+        'LocationBias',
+        'Dropdown'
```

Comment:
I think `Dropdown` could go higher in this order. but, it might be good to ask product where they would like it placed because I think they've structured their documentation to follow the ordering here

Reply:
Asked Tom about it; he said we should keep it at the bottom since no one uses it outside of `FilterSearch` and `SearchBar` yet